driver/crdb.go

```diff
@@ -150,6 +150,65 @@ func (d *CockroachDBDriver) TableNames(schema string, whitelist, blacklist []str
 	return names, nil
 }
 
+// ViewNames connects to the CockroachDB database and
+// retrieves all view names from the information_schema where the
+// view schema is schema. It uses a whitelist and blacklist.
+func (d *CockroachDBDriver) ViewNames(schema string, whitelist, blacklist []string) ([]string, error) {
+	var names []string
+
+	query := fmt.Sprintf(`select table_name from information_schema.tables where table_schema = $1 and table_type = 'VIEW'`)
```

Comment:
Any reason this needs to be `fmt.Sprintf`?

Reply:
Probably not, it's just how it was handled in existing code (`ViewNames` is basically just a copy/modification of `TableNames` which already worked this way)